odoo/addons/base/models/ir_demo.py

```diff
@@ -13,9 +13,15 @@ class IrDemo(models.TransientModel):
 
     @assert_log_admin_access
     def install_demo(self):
+        original_company_country_code = self.env.company.country_code
+
         force_demo(self.env)
+
+        targetCompany = self.env['res.company'].search([('country_code', '=', original_company_country_code)])
```

Comment:
no need of the intermediate variable `original_company_country_code` IMO, also add a limit=1 in the search

Reply:
It is actually needed because after the `force_demo(self.env)`, the country of the base company is set to US as it loads the demo_date linked to the generic coa. I need to save which country was set before this changed in order to jump to the right company afterwards.

Reply:
You're right :D sry